_authors/osci_2023/rahulsiddharth.markdown

```diff
@@ -0,0 +1,6 @@
+---
+name: Rahul Siddharth
+github: 908rahul
+---
+
+**<Rahul Siddharth>** <is a freshman at Computer Science at Purdue University that enjoys and is interested in C++!>
```

Comment:
```suggestion
**Rahul Siddharth** is a freshman at Computer Science at Purdue University that enjoys and is interested in C++!
```

Reply:
I think this is on my previous branch, I created a new one since I went down a rabbit hole in the first one. In the second one I believe It is the correct format with a signoff